### PR TITLE
sites that require cleaning

### PR DESCRIPTION
**Pull request by wout-codes**
These sites requires cleaning:
```
https://zeefiles.download/ce53d69da7059421
https://uploader.link/ub03m3zowh90
https://uploadflix.org/ug97mq5vfhmj
https://uploadraja.com/2gvegfkd8wrr - site changes code every hour
https://userupload.net/brneanmcz38f
```

https://github.com/AdguardTeam/AdguardFilters/issues/74995

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -254,6 +254,10 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+uploader.*##a[href*="https://shopcart4me.com"]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:257]: Why did you use `*`? Please provide some examples.
Why not `uploader.link##a[href^="https://shopcart4me.com"]`?

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -254,6 +254,10 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
...
+uploadflix.*##a[href^="https://t.co/"]
+uploadraja.*##a[target="_blank"]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:259]: Too wide rules. Why not `uploadflix.org,uploadraja.com##center > a[target="_blank"] > img`?

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -254,6 +254,10 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
...
+userupload.*##div[class^="ads mb-4"]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:260]: Why not `userupload.net##.ads`?

File: EnglishFilter/sections/general_extensions.txt [hunk 4]
```diff
@@ -229,6 +229,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! pop-ups in uploader.*, zeefiles.*
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:232]: Please provide some examples for using `*`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/general_extensions.txt (hunk 4 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:232]: I didn't look up but I think they have more domains.

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: I didn't look up but I think they have more domains.

File: EnglishFilter/sections/general_extensions.txt (hunk 4 above)
Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:232]: > I didn't look up but I think they have more domains.

That's not enough. We need real examples.
For this PR just use original domains without `*`.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:232]: Ok.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt]: @zloyden Swapped all.

File: EnglishFilter/sections/specific.txt [hunk 5]
```diff
@@ -270,6 +270,9 @@ rule34.xxx###halloween
 extramovies.*##.ad_btn1
 extramovies.*##a[href="/dlbutton.php"]
 extramovies.*##body > a[href]#theme_back
+uploader.link##a[href^="https://shopcart4me.com"]
+uploadflix.org,uploadraja.com##center > a[target="_blank"] > img
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:274]: We have a common rule `##center > a[target="_blank"] > img`. Add those domains to this rule.

File: EnglishFilter/sections/specific.txt [hunk 6]
```diff
@@ -270,6 +270,9 @@ rule34.xxx###halloween
...
+userupload.net##.ads
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:275]: The same about the `##.ads`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 6 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt]: Done.

File: EnglishFilter/sections/specific.txt (hunk 5 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt]: Grouped some domains and added these ones too.

**Review by zloyden — APPROVED** [review]

**Comment by wout-codes** [comment]
@zloyden Please close this issue, #74995.

**Pull request merged**